backend/mora/service/employee.py

```diff
@@ -483,14 +483,33 @@ def terminate_employee(employee_uuid):
         )
     ]
 
+    trigger_dict = {
+        'role_type': mapping.EMPLOYEE,
+        'event_type': Trigger.Event.ON_BEFORE,
+        'request': request,
+        'request_type': handlers.RequestType.TERMINATE,
+        'uuid': employee_uuid,
+        'employee_uuid': employee_uuid
+    }
+
+    if not flask.request.args.get('triggerless'):
+        Trigger.run(trigger_dict)
+
     for handler in request_handlers:
         handler.submit()
 
+    trigger_dict["event_type"] = Trigger.Event.ON_AFTER
+    trigger_dict["result"] = result = flask.jsonify(employee_uuid)
```

Comment:
I almost missed the assignment to `result` — perhaps either move it to the front of the line or a separate line?

Reply:
done - det er en drippepind, sådan en